plugin/test/unit/helpers/draft.config.spec.ts

```diff
@@ -0,0 +1,198 @@
+      const expectedFileName = `${publishDir}/${mockedDatastoreFile}`
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockCopySync).toHaveBeenCalledWith(
+        expectedDataFile,
+        expectedFileName,
+      )
+    })
+
+    it('should create expected metadata file in cached directory', async () => {
+      const expectedFileName = `${cacheDir}/dataMetadata.json`
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockEnsureFileSync).toHaveBeenCalledWith(expectedFileName)
+    })
+
+    it('should write expected metadata payload in cached directory', async () => {
+      const expectedFileName = `${cacheDir}/dataMetadata.json`
+      const expectedPayload = {
+        fileName: mockedDatastoreFile,
+      }
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockWriteJSON).toHaveBeenCalledWith(
+        expectedFileName,
+        expectedPayload,
+      )
+    })
+  })
+
+  describe('when previous datastore file does not exist', () => {
+    const mockedUuid = 'fake-uuid'
+    const expectedUuidFileName = `data-${mockedUuid}.mdb`
+
+    beforeEach(() => {
+      mockReadJSON.mockResolvedValue(null)
+      mockReaddir.mockResolvedValue([])
+      mockUuidv4.mockReturnValue(mockedUuid)
+    })
+
+    it('should create expected filename in published directory', async () => {
+      const expectedFileName = `${publishDir}/${expectedUuidFileName}`
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockEnsureFileSync).toHaveBeenCalledWith(expectedFileName)
+    })
+
+    it('should copy cached datastore file to published directory', async () => {
+      const expectedDataFile = join(`${cacheDir}/data/datastore/data.mdb`)
+      const expectedFileName = `${publishDir}/${expectedUuidFileName}`
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockCopySync).toHaveBeenCalledWith(
+        expectedDataFile,
+        expectedFileName,
+      )
+    })
+
+    it('should create expected metadata file in cached directory', async () => {
+      const expectedFileName = `${cacheDir}/dataMetadata.json`
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockEnsureFileSync).toHaveBeenCalledWith(expectedFileName)
+    })
+
+    it('should write expected metadata payload in cached directory', async () => {
+      const expectedFileName = `${cacheDir}/dataMetadata.json`
+      const expectedPayload = {
+        fileName: expectedUuidFileName,
+      }
+
+      await createMetadataFileAndCopyDatastore(publishDir, cacheDir)
+
+      expect(mockWriteJSON).toHaveBeenCalledWith(
+        expectedFileName,
+        expectedPayload,
+      )
+    })
+  })
+})
+/* eslint-enable max-lines-per-function */
+/* eslint-enable max-nested-callbacks */
+/* eslint-enable  @typescript-eslint/no-explicit-any */
```

Comment:
Remove once fixed.

Reply:
As far as I know, and happy to be corrected, I don't think there's a need to enable rules at the end of a file. Should be okay to remove these and just keep the `eslint-disable` at the top

Reply:
Our linting rules insist that disabled rules are re-enabled